spec/classes/foreman_proxy__plugin__remote_execution__ssh_spec.rb

```diff
@@ -65,5 +70,26 @@
     end
 
     it { should_not contain_exec('generate_ssh_key') }
+    it { should_not contain_file('/root/.ssh') }
```

Comment:
no, since `generate_ssh_key` is false, I think this actually verifies what you asked for in main comment. The $install_key already is in $generate_keys block https://github.com/ares/puppet-foreman_proxy/blob/c3b40e9d019dcef6fd789995d4bfabb420ce827b/manifests/plugin/remote_execution/ssh.pp#L59-L87